### PR TITLE
Sorted by UI: remove italic + change icon

### PR DESCRIPTION
**Pull request by agnesstenlund**
### Before
CleanShot 2023-01-12 at 14 38 57

### After
CleanShot 2023-01-12 at 14 39 43

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1203713947945019) by [Unito](https://www.unito.io)

## Design recommendations
* **Remove italic:** You rarely see italic used on buttons, menu items, product labels. It's usually preferred to use lighter colors and smaller font sizes if you want to deemphasise something.
* **Change icon:** The `sort` icon seems more suitable than the settings cog (I prefer the up down icon even more (see Asana screenshot below) but there was none in the Mui library so I started with the one they call `Sort`)
CleanShot 2023-01-12 at 14 48 36

### CODE REVIEW

File: packages/lesswrong/components/icons/SettingsButton.tsx [hunk 1]
```diff
@@ -33,11 +34,11 @@ const SettingsButton = ({classes, className, onClick, showIcon=true, label=""}:
 }) => {
   if (label) {
     return <span className={classes.iconWithLabelGroup} onClick={onClick}>
-      {showIcon && <Settings className={classNames(classes.icon, classes.iconWithLabel, className)}/>}
+      {showIcon && <Sort className={classNames(classes.icon, classes.iconWithLabel, className)}/>}
       <span className={classes.label}>{ label }</span>
     </span>
   }
-  return <Settings className={classNames(classes.icon, className)} onClick={onClick}/>
+  return <Sort className={classNames(classes.icon, className)} onClick={onClick}/>
```
Comment (Will-Howard) [inline packages/lesswrong/components/icons/SettingsButton.tsx:41]: I'm a bit concerned about this icon being too close to the one we use for shortform, there is a material UI icon called (confusingly) SwapVert which has the little arrows which might be better
Screenshot 2023-01-12 at 17 49 38

**Review by Will-Howard — COMMENTED** [review]
I'm loving all the PRs from non-developers, soon we won't have to do anything 😌

File: packages/lesswrong/components/icons/SettingsButton.tsx (hunk 1 above)
Comment (agnesstenlund) [reply packages/lesswrong/components/icons/SettingsButton.tsx:41]: Oh brilliant! Will update to that, thanks

**Comment by jpaddison3** [comment]
This is a UI PR affecting LW, and (although minor), I would err on the side of getting their review as well. We either get them to approve it or we can make the change specific to the EA Forum only. I would recommend, while you're getting started, to spend "too much time" on communicating about UI with LW.

**Comment by jimrandomh** [comment]
Lgtm

**Comment by jpaddison3** [comment]
I happen to be on a call with @jimrandomh now though, and he says... ^^^ that

**Comment by jimrandomh** [comment]
Actually, looks like this changes the icon in a bunch of places at once, some of which don't make sense with the sorting icon. For example on post pages:
Screenshot 2023-01-12 at 11 13 00

**Comment by jpaddison3** [comment]
No worries about that Agnes, that's our job to catch, and we did. Do you want to fix it, or would you like to hand it off to a dev?

**Comment by agnesstenlund** [comment]
Nice catch, I thought it would make sense in all other instances too but you're totally right it doesn't 😅  I don't want to create more work for you so I'll see if I can fix it myself first🙏

File: packages/lesswrong/components/icons/SettingsButton.tsx [hunk 2]
```diff
@@ -6,22 +6,23 @@ import Settings from '@material-ui/icons/Settings';
...
+    marginRight: 4,
+    marginLeft: 4,
   },
   label: {
     ...theme.typography.body2,
     fontSize: 14,
     color: theme.palette.grey[600],
-    fontStyle: "italic"
-  }
+  },
```
Comment (agnesstenlund) [inline packages/lesswrong/components/icons/SettingsButton.tsx:25]:
```suggestion
  }
```

**Comment by agnesstenlund** [comment]
@jpaddison3 I've made some updates to catch those other places–could you review again? 😇

**Comment by jpaddison3** [comment]
[No action needed] I'm very confused where the asana task for this went. (We're supposed to have [Unito](https://unito.io/) create asana tasks for us for Pull Requests.) I dunno why this one didn't.

**Comment by jpaddison3** [comment]
Oh wait I found it.

**Review by jpaddison3 — APPROVED** [review]
LGTM

**Pull request merged**